tests/sentry/integrations/github/test_integration.py

```diff
@@ -120,7 +127,12 @@ def _stub_github(self):
             "archived": {
                 "archived": True,
             },
+            "xyz": {
+                "full_name": "Test-Organization/xyz",
+                "default_branch": "master",
+            },
         }
+        self.repositories = repositories
```

Comment:
This allows me to use `self.repositories` without refactoring everywhere where `repositories` is used.